gnd/src/main/java/com/google/android/gnd/ui/home/BottomSheetChromeBehavior.java

```diff
@@ -39,6 +40,10 @@ public BottomSheetChromeBehavior(Context context, AttributeSet attrs) {
   @Override
   protected void onSheetScrolled(
       CoordinatorLayout parent, ViewGroup layout, SheetSlideMetrics metrics) {
+    // Slide feature sheet header up and down together with bottom sheet.
+    View header = parent.findViewById(R.id.feature_sheet_header);
+    header.setY(metrics.getTop() -  header.getHeight());
```

Comment:
@dturner this is what I had in mind to simplify obscuring the sheet header behind the toolbar; we can split it out completely into the parent FrameLayout and control its position here.